### DISCUSSION TITLE
Howto use Atmel ICE as upload method

### DISCUSSION THREAD

**Discussion by PerennialNovice** [Sloeber/arduino-eclipse-plugin#1434] [Q&A] [answered]
I have a board with broken bootloader that I still want to use.
I used to program it with the Atmel ICE in the Arduino IDE.

Now I try to achieve this with sloeber as well, but simply choosing Atmel ICE (AVR) as upload protocol does not seem to be sufficient.
I don't know, but should avrdude be summoned to do the upload?

> Starting upload

> 
> Uploading project "PowerSupply" with "avrdude"
> 
> 
> Launching: C:\Sloeber\arduinoPlugin\packages\arduino\tools\avrdude\6.3.0-arduino17/bin/avrdude -CC:\Sloeber\arduinoPlugin\packages\arduino\tools\avrdude\6.3.0-arduino17/etc/avrdude.conf -patmega2560 -cwiring -PCOM13 -b115200 -D -Uflash:w:C:\Users\xxxXXXxxx\Documents\sloeber_workspace\PowerSupply\Release/PowerSupply.hex:i 
> Output:
> avrdude: stk500v2_ReceiveMessage(): timeout
> avrdude: stk500v2_ReceiveMessage(): timeout
> avrdude: stk500v2_ReceiveMessage(): timeout
> avrdude: stk500v2_ReceiveMessage(): timeout

This looks exactly as if I had chosen the standard upload method

Have I missed some setting?

**Comment by jantje** [comment, marked as answer]
Sloeber behaviour has changed in the nightly.
In the nightly you need to select upload using programmer (just like in the arduino IDE)
In the curent latest release the upload should do the trick

**Comment by PerennialNovice** [comment]
Thank you very much, that's the bit of info I needed :)
